### DISCUSSION TITLE
Get Altitude in meters from N19E095.hgt file in flutter_map

### DISCUSSION THREAD

**Discussion by yannaung2019** [fleaflet/flutter_map#1254] [Ideas]
Hello all I'm developing flutter offline map app. I already get Latitude and Longitude when i tap on a map. But i donn't get altitude and elevation when i tap on map. I have .hgt file and how to get altitude in  meters from this file in flutter.I'm already search on internet but not yet found solution so that pls answer me if you known . thz

**Comment by ibrierley** [comment]
There isn't any way to get elevation. You would need some other service to give you that info like Google Elevation API.

**Reply by yannaung2019** [reply]
Thank Sir for your reply but i'm creating offline flutter_map app so that this api  cann't help me :(

**Comment by ibrierley** [comment]
Ah sorry, I didn't read that you have a .hgt file...what is in that ?

**Reply by yannaung2019** [reply]
Thank for your reply sir . It was SRTM for map and if we want to known altitude of a location we used this file in Locus pro App.

**Comment by ibrierley** [comment]
Wondering if something like https://github.com/aatishnn/srtm-python/blob/master/srtm.py would help you.

**Reply by yannaung2019** [reply]
Thank you for your reply sir . i will try it.

**Comment by JaffaKetchup** [comment]
Maybe this could help you? 
https://gis.stackexchange.com/a/43756

**Reply by yannaung2019** [reply]
Thank you for your reply sir.
